src/type-coercions.md

```diff
@@ -195,10 +195,14 @@ r[coerce.unsize]
 
 r[coerce.unsize.intro]
 The following coercions are called `unsized coercions`, since they
-relate to converting sized types to unsized types, and are permitted in a few
+relate to converting types to unsized types, and are permitted in a few
```

Comment:
```suggestion
relate to converting sized types to unsized types or converting from one unsized type to another, and are permitted in a few
```

(more clarity, same meaning)

Reply:
For me your proposed change kinda flushes the cache, I had to re-read it 4 times before I figured out what it means 💀 (then again, maybe I shouldn't be reading it at 3am)

Since it's the same meaning I'd think that more concise is better? Do you disagree?